WalletWasabi/Crypto/ZeroKnowledge/StrobeFlags.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WalletWasabi.Crypto.ZeroKnowledge
+{
+	[Flags]
+	public enum StrobeFlags
```

Comment:
i think this should stay private since it shouldn't be used... perhaps as an inner class of State?